### DISCUSSION TITLE
Github commits are not deploying to vercel

### DISCUSSION THREAD

**Discussion by BraLaryea** [vercel/vercel#8685] [Help] [answered]
My github commits are not deploying to vercel, the commit has no issues on githut and no build issues, it simply isnt deploying

**Comment by BraLaryea** [comment, marked as answer]
Was a general issue, seems to be fixed now

**Reply by amyegan** [reply]
Some people may still be facing this issue. You can find related updates at https://www.vercel-status.com/incidents/8bb26zh7xjsf

**Reply by anthonychung14** [reply]
thank you! I noticed that my build has been marked as "queued". should we standby for these to resolve or cancel the "stuck" build?

**Reply by amyegan** [reply]
We have a pretty large queue that flooded in all at once because of some upstream issues. I would standby for the queued build 🙇

**Reply by BraLaryea** [reply]
> Some people may still be facing this issue. You can find related updates at https://www.vercel-status.com/incidents/8bb26zh7xjsf
Oh ok


> We have a pretty large queue that flooded in all at once because of some upstream issues. I would standby for the queued build 🙇
Noted, noticed the same thing after my commits started building once again

**Reply by amyegan** [reply]
The team is continuing to do everything they can to improve the situation. Thank you for your patience as we work through this!

You can find the latest updates on our [status page](https://www.vercel-status.com/)
- Builds queued: https://www.vercel-status.com/incidents/qgxyw00n8zz6
- Git commits not building: https://www.vercel-status.com/incidents/8bb26zh7xjsf

---

Update: This incident is resolved. All systems are operational. If you are experiencing platform problems, please [start a new discussion](https://github.com/vercel/community/discussions/new) or [submit a case](https://vercel.com/help#issues) to the Customer Success Team.
